src/CMakeLists.txt

```diff
@@ -148,8 +148,11 @@ configure_file(${CMAKE_SOURCE_DIR}/docs/desktopEntry/package/flameshot.desktop
 configure_file(${CMAKE_SOURCE_DIR}/docs/appdata/flameshot.metainfo.xml
                ${CMAKE_CURRENT_BINARY_DIR}/share/metainfo/flameshot.metainfo.xml COPYONLY)
 
-configure_file(${CMAKE_SOURCE_DIR}/docs/bash-completion/flameshot
-               ${CMAKE_CURRENT_BINARY_DIR}/share/bash-completions/completions/flameshot COPYONLY)
```

Comment:
Possible typo here. I don't know much about bash, but it is `/usr/share/bash-completion` without s on my systems (and in comment above).